docs/proposals/1374-mc-inference-gateways/README.md

```diff
@@ -0,0 +1,165 @@
+
+#### InferencePoolImport
+
+A new API that mirrors ServiceImport from the MCS API. This allows anyone in a connected cluster to reference a Multi-Cluster InferencePool, even if the local cluster does not have a local InferencePool. In the context of Gateway API, that means that a Gateway could be configured to reference an InferencePoolImport, even if that cluster did not contain an InferencePool.
+This API will be used almost exclusively for tracking endpoints, but unlike MCS, we actually have two distinct sets of endpoints that we could track:
+
+1. Endpoint Pickers
+1. Model Server Endpoints
+
+## Implementation Details
+
+In the happy path, the only type of endpoint that a Gateway would need to know about is Endpoint Pickers. Ultimately, each Gateway will be sending requests to Endpoint Pickers, and then following the directions of that Endpoint Picker. As long as an Endpoint Picker is available, there’s no need to actually propagate the model server endpoints.
+
+### Failure Mode
+
+If the Endpoint Picker is unavailable and the failure mode is configured as “FailOpen”, we could take one of several approaches:
+
+#### Honor FailOpen configuration
+
+This seems to require the Gateway to be aware of at least some model server endpoints, which requires more endpoint propagation.
+
+#### Fail over to other cluster/Endpoint Picker
+
+In a world where there are multiple clusters/Endpoint Pickers to choose from, it may be desirable to fail over to another cluster. Ultimately if all Endpoint Pickers are unavailable, we may end up back at the same problem though of needing to be aware of model server endpoints.
+
+#### Consider FailOpen “Extended” support for multi-cluster
+
+Given the potential complexity of supporting a FailOpen mode for multi-cluster, we could consider this “Extended” or optional support.
+
+### Cluster/Endpoint Picker Selection
+
+It’s likely that each Gateway implementation will have some different logic here, but there will likely be at least two common paths here:
+
+#### Metrics from model server endpoints
+
+In the case where a Gateway is aware of all model server endpoints, it could theoretically also track metrics for each of these endpoints.
+
+#### Metrics from Endpoint Picker
+
+Since Gateways are ultimately deciding which Endpoint Picker to send traffic to, it could make sense for Endpoint Pickers to report back load/utilization data to the Gateway to help inform that decision. (This would reflect the utilization of model server Pods within the local InferencePool managed by each EPP).
```

Comment:
EPP already exposes InferencePool metrics. It will be up to the implementation on how to use these metrics to make a routing decision.

Reply:
I think what @danehans is referring to here is that each Endpoint Picker already exposes [metrics](https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/7fcce71c0b25dd826e50790b2a7ae0d133ead53d/pkg/epp/metrics/metrics.go#L144) like `average_queue_size` and `average_kv_cache_utilization` that could be helpful for a Gateway when determining which cluster a request should be routed to.